src/construction.cpp

```diff
@@ -2568,22 +2598,30 @@ build_reqs get_build_reqs_for_furn_ter_ids(
                     if( ( pre_build.category == construction_category_DECORATE ) != allow_decorate ) {
                         continue;
                     }
-                    if( pre_build.category == construction_category_REPAIR ) {
+                    if( pre_build.category == construction_category_REPAIR ||
+                        pre_build.category == construction_category_DECONSTRUCT ) {
```

Comment:
This should have been in #77163, along with the equivalent check ~50 lines down.